bazel/python.bzl

```diff
@@ -1,3 +1,5 @@
+load("@bazel_skylib//lib:paths.bzl", "paths")
```

Comment:
This file was added from https://github.com/ray-project/ray/commit/c74886a55e81dc8f4150094bfec69d875f0b365d to fix CI test for notebooks.